base/float.jl

```diff
@@ -530,7 +530,8 @@ abs(x::Float64) = abs_float(x)
 """
     isnan(f) -> Bool
 
-Test whether a floating point number is not a number (NaN).
+Test whether a number value is a NaN, an indeterminate value which is neither an infinity
```

Comment:
```suggestion
Test whether a number value is a `NaN`, an indeterminate value which is neither an infinity
```

Reply:
I did not do this because ˋNaNˋ would be Julia code and then explicitly the ˋFloat64ˋ value ˋNaNˋ

Reply:
Ah, didn't realise NaN was Float64. :+1: